bitcoin/src/bip32.rs

```diff
@@ -31,11 +31,11 @@ const VERSION_BYTES_TESTNETS_PRIVATE: [u8; 4] = [0x04, 0x35, 0x83, 0x94];
 
 /// The old name for xpub, extended public key.
 #[deprecated(since = "0.31.0", note = "use xpub instead")]
```

Comment:
You need to update this message to

```suggestion
#[deprecated(since = "0.31.0", note = "use Xpub instead")]
```